tests/cypress/support/welcome_and_header.js

```diff
@@ -97,6 +103,11 @@ export const userMenu = {
         cy.url().should('equal', Cypress.config().baseUrl + '/search/')
         cy.visit('/multicloud/welcome')
     },
+    openCreate: () => {
+        cy.get('[aria-label="create-button"]').click()
+        cy.url().should('equal', 'https://localhost:3000/multicloud/k8s/all-namespaces/import')
```

Comment:
If you run this against real cluster, the url won't be localhost:3000. So I think this will fail.